examples/helper.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+lib = File.expand_path('lib', __dir__)
+$LOAD_PATH.unshift(lib) if !$LOAD_PATH.include?(lib)
+
+require 'colored'
+
+require_relative '../lib/oppen'
+
+# Display helpers.
+class String
+  def title          = green
+  def sub_title      = cyan
+end
```

Comment:
I don't like this. No real justification to extend `String`. We're only using colors with `puts`, so let's write them as functions:

```ruby
def title t = puts t.green
def subtitle t = puts t.cyan
```

And note that I removed the underscore.